examples/mongodbatlas_advanced_cluster/asymmetric-sharded-cluster/v1.x.x/main.tf

```
@@ -0,0 +1,83 @@
+provider "mongodbatlas" {
+  public_key  = var.public_key
+  private_key = var.private_key
```

```suggestion
  client_id     = var.mongodbatlas_client_id
  client_secret = var.mongodbatlas_client_secret
```

SA is already in master, same for the other occurences

---

as discussed in slack this will be done as a follow up